src/boot/efi/stub.c

```diff
@@ -21,6 +21,15 @@
 /* magic string to find in the binary image */
 _used_ _section_(".sdmagic") static const char magic[] = "#### LoaderInfo: systemd-stub " GIT_VERSION " ####";
 
+struct addon_additions {
```

Comment:
"addon_additions" sounds weird, because it has "add" twice in it?

maybe addon_components? or addon_info, or addon_data, or just addon?

Reply:
addon_components sounds good to me, I am only interested into reflecting side effects caused by the components, hence the additions